apps/pie-storybook/stories/pie-modal.stories.ts

```diff
@@ -144,6 +152,7 @@ const BaseStoryTemplate = (props: ModalProps): TemplateResult => {
             ?isOpen="${isOpen}"
             dir="${dir}"
             size="${size}"
+            .leadingAction="${leadingAction}"
```

Comment:
Just for my learning: what is the `.` here for?

Reply:
There's a bit about it here and a reference to the doc 👍 https://github.com/justeattakeaway/pie/pull/632#discussion_r1268568805